ibrdtn/daemon/tests/unittests/FakeDatagramService.cpp

```diff
@@ -105,7 +105,7 @@ size_t FakeDatagramService::recvfrom(char *buf, size_t length, char &type, char
 		seqno = msg.seqno;
 		address = msg.address;
 		ret = msg.data.size();
-		::memcpy(buf, &msg.data[0], (ret > length) ? length : ret);
+		if (ret > 0) ::memcpy(buf, &msg.data[0], std::min(ret, length));
```

Comment:
Header <algorithm> should be included in order to use std::min().

Reply:
Oh, I just noticed that github stripped the header I mentioned.
So again, std::min() requires to include "algorithm". Please add it to the list on top of the cpp file.